inc/shortcodes.php

```diff
@@ -181,11 +181,14 @@ function sell_media_checkout_shortcode() {
 	if ( ! empty( $cart_items ) ) :
 	?>
 	<div id="sell-media-checkout-cart">
-		<ul class="sell-media-cart-items">
+    <div class="left-sell-coulmn">
```

Comment:
Please use existing CSS classes/styling. There are grid classes for this. Please also remove any CSS that was added.

Reply:
Removed